notes/src/index.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import render from 'react';
+import { Component } from 'react'
+import NotesDefaultSource from './components/NotesDefaultSource';
+
+export default class App extends Component { // Step 4: Write a class called App, extending Component.
+  state = { // Step 7: Inside the constructor function, create a state on this, and set it to an object.       // Step 8: Inside the object, set a key to movies, and the value to an array of movies
+  };
+}
```

Comment:
not sure why this is here since you already have App being imported from a separate file